enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2054,6 +2082,50 @@ func (ws *workspace) fetch(ctx context.Context, remoteURL string, refs []string,
 	return nil
 }
 
+// Writes a wrapper script that invokes the ci_runner with the bazel_wrapper subcommand.
+// Also adds it to the PATH so it will be invoked whenever `bazel` or `bazelisk` are called.
+// The wrapper script adds a startup option for the custom ci_runner .bazelrc to
+// all bazel commands.
+func (ws *workspace) writeBazelWrapperScript() error {
+	wrapperDir := filepath.Join(ws.rootDir, "wrappers")
+	for _, c := range []string{bazelBinaryName, bazeliskBinaryName} {
+		wrapperPath := filepath.Join(wrapperDir, c)
+		_, err := os.Stat(wrapperPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				if err := os.MkdirAll(filepath.Dir(wrapperPath), 0755); err != nil {
+					return err
+				}
+			} else {
+				return err
+			}
+		}
+
+		f, err := os.OpenFile(wrapperPath, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0755)
+		if err != nil {
+			return status.InternalErrorf("Failed to open %s for writing: %s", wrapperPath, err)
+		}
+		defer f.Close()
+
+		contents := `
+#!/usr/bin/env sh
+BAZEL_WRAPPER_MODE=1 BAZEL_BIN=` + *bazelCommand + ` CI_RUNNER_ROOT=` + ws.rootDir + ` ` + os.Getenv("BUILDBUDDY_CI_RUNNER_ABSPATH") + ` "$@"
```

Comment:
nit, since paths can contain spaces, use `fmt.Sprintf("%q", path)` for each path so that they are properly quoted and escaped